### PR TITLE
Create the bubble charts section

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #300.

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/bubble.md [hunk 1]
```diff
@@ -1,3 +1,178 @@
...
+
+Here is a small example of this:
+
+```py
+data = {
+    "x": [1, 2, 3],
+    "y": [1, 2, 3],
+    "Colors": ["blue", "green", "red"],
+    "Sizes": [20, 40, 30],
+    "Opacity": [1, .4, 1]
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/bubble.md]: Opacities ? bizarre ...

File: docs/manuals/gui/viselements/charts/scatter.md [hunk 2]
```diff
@@ -10,11 +10,12 @@ In order to create scatter charts with Taipy, you need to set the
 
 | Name            | Value           | Notes   |
 | --------------- | ------------------------- | ------------------ |
+| [*type*](../chart.md#p-type)      | `scatter`          | Default value  |
 | [*mode*](../chart.md#p-mode)      | `markers`          |  |
 | [*x*](../chart.md#p-x)            | x values           |  |
 | [*y*](../chart.md#p-y)            | y values           |  |
-| [*marker*](../chart.md#p-marker)  | dictionary  | `size` can be an integer or a column name in *data*.<br/>`color` can be a color value or a column name in *data*.<br/>`symbol` must be a predefined symbol name or a column name in *data*.<br/>`opacity` can be a opacity value or a column name in *data*.|
-| [*layout*](../chart.md#p-layout) | dictionary | Global layout settings. |
+| [*marker*](../chart.md#p-marker)  | dictionary  | `size` can be set to an integer value.<br/>`color` can be set to a color value.<br/>`symbol` can be set to a predefined symbol name.<br/>`opacity` can be set to an opacity.  |
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/scatter.md]: opacity (float value <= 1) ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/manuals/gui/viselements/charts/bubble.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/bubble.md]: However correct. And it clarifies the distinction between the property name and the column name.

File: docs/manuals/gui/viselements/charts/scatter.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/scatter.md]: Added.

**Pull request merged**